evm/src/main/java/org/hyperledger/besu/evm/frame/Memory.java

```diff
@@ -179,7 +180,8 @@ int getActiveBytes() {
    *
    * @return The current number of active words stored in memory.
    */
-  int getActiveWords() {
+  @VisibleForTesting
+  public int getActiveWords() {
```

Comment:
don't think this needs to be public any more

Reply:
It actually needs; other tests break otherwise.